src/style.css

```diff
@@ -1,3 +1,73 @@
-body {
+body,
+ul,
+li {
   margin: 0;
+  padding: 0;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  --color: #00acdc;
```

Comment:
are you realy need this rules for body,ul,li?

Reply:
I think this one is a better option compared to zeroing out padding / margin for ul and li to avoid unnecessary code. Otherwise without zeroing them the test will fail.